### DISCUSSION TITLE
Postgres calling PROCEDURE in NestJS

### DISCUSSION THREAD

**Discussion by AgentSource** [prisma/prisma#12595] [Q&A]
This feels like more of a question then a bug... but I could be wrong — I'm struggling to CALL a PROCEDURE 

My PROCEDURE: 
```sql
CREATE OR REPLACE PROCEDURE refresh_mat_view()
LANGUAGE plpgsql
AS $$
BEGIN
REFRESH MATERIALIZED VIEW materialized_created_by;
END $$;
```

which works perfectly when I run it in the console however when I call execute this call

```typescript 
client.$executeRaw`CALL refresh_mat_view();` 
```

I get this error back

```
Raw query failed. Code: `42883`. Message: `db error: ERROR: procedure refresh_mat_view() does not exist
HINT: No procedure matches the given name and argument types. You might need to add explicit type casts.`
    at Object.request (/node_modules/@prisma/client/runtime/index.js:39818:15)
    at Proxy._request (/node_modules/@prisma/client/runtime/index.js:40649:18)
```
    
I was able to list all  PROCEDURE on the DB and it does indeed show up

```sql
SELECT  nspname, proname
FROM    pg_catalog.pg_namespace
JOIN    pg_catalog.pg_proc
ON      pronamespace = pg_namespace.oid
WHERE   nspname = 'public' AND proname = 'refresh_mat_view'
```

Will in turn return => `[ { nspname: 'public', proname: 'refresh_mat_view' } ]`

I have also attempted to use `$queryRaw`, `$queryRawUnsafe`, `executeRawUnsafe` and get the same error

I'm assuming I'm missing something or there is a different method I could be using to execute a PROCEDURE

Any help would be awesome — thanks!

Side note I referenced this issue to attempt to debug more => #9868

**Comment by nurul3101** [comment]
Hey @AgentSource 👋 
Could you please add your use case and provide these reproduction details in the [GitHub Issue](https://github.com/prisma/prisma/issues/9868) that you have referenced.

It will help us in testing the improvements which we are currently doing for Raw Queries.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
